index.html

```diff
@@ -27,7 +27,138 @@
 <script type="module" src="/src/main.js"></script>
 <script>
   window.addEventListener('load', function () {
-    window.tido = new window.Tido();
+    window.tido = new window.Tido({
```

Comment:
Don't modify index.html pls

Reply:
okay, restored its content in a further commit